### PR TITLE
SHA-2369 : AIMS remove admin tools bits

### PR DESCRIPTION
**Pull request by alexbalmus**

### CODE REVIEW

File: share/src/main/resources/alfresco/site-webscripts/org/alfresco/components/console/usersgroups/users.get.properties [hunk 1]
```diff
@@ -2,6 +2,7 @@ label.title-search=User Search
 label.title-view=User Profile
 label.title-create=New User
 label.title-update=Edit User
+label.restricted-settings=Some settings are restricted by your organization.
```
Comment (cristina-sirbu) [inline share/src/main/resources/alfresco/site-webscripts/org/alfresco/components/console/usersgroups/users.get.properties:5]: Why do we need this?

File: pom.xml [hunk 2]
```diff
@@ -73,7 +73,7 @@
 
         <tomcat.default.alfresco.port>8080</tomcat.default.alfresco.port>
         <tomcat.default.alfresco.ssl.port>8443</tomcat.default.alfresco.ssl.port>
-        <tomcat.default.share.port>8081</tomcat.default.share.port>
+        <tomcat.default.share.port>8181</tomcat.default.share.port>
```
Comment (bogdanioan-gabor) [inline pom.xml]: @alexbalmus Should we change it?

File: share/src/main/java/org/alfresco/web/scripts/UsersWebScript.java [hunk 3]
```diff
@@ -0,0 +1,60 @@
+/*
```
Comment (bogdanioan-gabor) [inline share/src/main/java/org/alfresco/web/scripts/UsersWebScript.java]: @alexbalmus I find the name of this webscript confusing. I can't see this web script being related to users.

File: share/src/main/resources/share-config.properties [hunk 4]
```diff
@@ -1,8 +1,8 @@
 #Properties for AIMS 
-aims.config.enabled=false
+aims.config.enabled=true
```
Comment (bogdanioan-gabor) [inline share/src/main/resources/share-config.properties]: @alexbalmus Can you keep the aims.config.enabled false, and remove the authServerUrl.

File: share/src/main/webapp/components/console/users.js [hunk 5]
```diff
@@ -119,6 +119,8 @@
           */
          onLoad: function onLoad()
          {
+            var aimsEnabled =  typeof parent.options.aimsEnabled === "boolean" ?  parent.options.aimsEnabled : false;
```
Comment (bogdanioan-gabor) [inline share/src/main/webapp/components/console/users.js:122]: @alexbalmus Can't see the local variable aimsEnabled used anywhere.

File: share/src/main/resources/alfresco/site-webscripts/org/alfresco/components/profile/userprofile.get.properties [hunk 6]
```diff
@@ -9,3 +9,4 @@ label.photoimagesize=Your image will be resized to 64px by 64px.
 label.photonote=Changes to your photo image take effect immediately.
 button.editprofile=Edit Profile
 button.usedefault=Use Default
+label.restricted-settings=Some settings are restricted by your organization.
```
Comment (bogdanioan-gabor) [inline share/src/main/resources/alfresco/site-webscripts/org/alfresco/components/profile/userprofile.get.properties:12]: @alexbalmus Can we keep the label.restricted-settings option only in one value, set it in a globally scoped properties file.

**Pull request merged**